source/command-monitoring/tests/find.json

```diff
@@ -0,0 +1,239 @@
+          },
+          "sort": {
+            "_id": 1
+          },
+          "skip": 2,
+          "limit": 2
+        }
+      },
+      "expectations": [
+        {
+          "command_started_event": {
+            "command": {
+              "find": "values",
+              "filter": {
+                "_id": 1
+              },
+              "sort": {
+                "_id": 1
+              },
+              "skip": 2,
+              "limit": 2
+            },
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        },
+        {
+          "command_succeeded_event": {
+            "reply": [
+              {
+                "_id": 1,
+                "x": 11
+              }
+            ],
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        }
+      ]
+    },
+    {
+      "description": "A successful find event with a getmore",
+      "operation": {
+        "name": "find",
+        "arguments": {
+          "filter": {
+            "_id": {
+              "$gte": 1
+            }
+          },
+          "sort": {
+            "_id": 1
+          },
+          "batchSize": 3
+        }
+      },
+      "expectations": [
+        {
+          "command_started_event": {
+            "command": {
+              "find": "values",
+              "filter": {
+                "_id": {
+                  "$gte": 1
+                }
+              },
+              "sort": {
+                "_id": 1
+              },
+              "batchSize": 3
+            },
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        },
+        {
+          "command_succeeded_event": {
+            "reply": [
+              {
+                "_id": 1,
+                "x": 11
+              },
+              {
+                "_id": 2,
+                "x": 22
+              },
+              {
+                "_id": 3,
+                "x": 33
+              }
+            ],
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        },
+        {
+          "command_started_event": {
+            "command": {
+              "collection": "values",
+              "batchSize": 3
+            },
+            "command_name": "getMore",
+            "database_name": "command-monitoring-tests"
+          }
+        },
+        {
+          "command_succeeded_event": {
+            "reply": [
+              {
+                "_id": 4,
+                "x": 44
+              },
+              {
+                "_id": 5,
+                "x": 55
+              }
+            ],
+            "command_name": "getMore",
+            "database_name": "command-monitoring-tests"
+          }
+        },
+        {
+          "command_started_event": {
+            "command": {
+              "killCursors": "values"
```

Comment:
I assume you intentionally leave out the "cursors" field from here because there is no way to know in the test what the value of it will be?  If so, I think that's a mistake.  We need a way to specify a placeholder value that can be filled in by the test runner.